### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Chawki-Mbarki**
# ✨ Verify User's Email✨

👋 Hi there,

This pull request adds a JavaScript script to validate the user email input in the contact form. The script checks if the email is in lowercase and prevents the form from submitting if it is not. It also displays an error message in a styled paragraph with the correct email format.

## The script uses the following functions:

- formErrors(): returns an array of errors based on the email input.
- formValidation(): returns true or false based on the errors array and displays the error message if needed.
- (**Optional Requirement**): the error message will contain the user's email address but in lower case.

This feature improves the user experience and ensures that the email input is consistent and valid.

## Here is a screenshot 🖼️ of the final result:
formerror

Please review my code and tell me if there is anything that needs to change. Thank you! 😊

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -315,6 +323,30 @@ function displaySelectedImage(direction) {
   return null;
 }
 
+// return form errors
+function formErrors() {
+  /*  const name = document.querySelector("#name").value;
+      const message = document.querySelector("#message").value; dont need this for now */
```
Comment (youmari) [inline script.js]: - [x] it would be better to remove unused commented code to keep the file clean and pro

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @chawkimbarki , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Chawki-Mbarki) [reply script.js]: hey @youmari  👋 

thanks for giving me the chance to explain this.
🤷 that comment is for future use as explained: "don't need this for now".

Comment (youmari) [reply script.js]: @chawkimbarki alright then

**Review by ShadyShawkat — APPROVED** [review]
## Hi @chawkimbarki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**